Engine/ResourceManagement/Resources/Skeleton.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Skeleton.h"
+
+
+Skeleton::Skeleton(const char * const UID, const std::string & exported_file) : Resource(UID, exported_file)
+{
+
+}
+
+Skeleton::Skeleton(std::vector<Joint> && joints, const std::string & exported_file) :
+skeleton(joints),
+Resource("", exported_file)
+{
+	LoadInMemory();
+
+}
+void Skeleton::Save(Config& config) const
```

Comment:
Save and Load not implemented means that you can't save a scene with a Skeleton?

Reply:
Save and load here are for the resource, in the future resources will have importing options and this functions are for that. To save it load it in the scene you will have to create skeleton as a component or as I did as a bunch of gameobjects